test/functional/test_framework/mininode.py

```diff
@@ -1771,7 +1771,7 @@ def got_message(self, message):
                 self.messagemap[b'ping'] = msg_ping_prebip31
         if self.last_sent + 30 * 60 < time.time():
             self.send_message(self.messagemap[b'ping']())
-        logger.debug("Received message from %s:%d: %s" % (self.dstaddr, self.dstport, repr(message)))
+        logger.debug("Received message from %s:%d: %s" % (self.dstaddr, self.dstport, repr(message)[:500]))
```

Comment:
I've added this in the latest fixup commit. I also added a _log_message() method to avoid code duplication.